### PR TITLE
My001

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Two new edited items from MY collection. 
For MY-001, how can I show relations between the fragments of the text scattered in the volume due to misplaced leaves?

### CODE REVIEW

File: ES/ESmy001.xml [hunk 1]
```diff
@@ -30,324 +30,712 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                          <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡</hi> ወመንፈስ፡ ቅዱስ፡ <hi rend="rubric">ዝንቱ፡ ግብሮሙ፡</hi> ለክልኤቱ፡
+                          አርድእት፡ ቡ<hi rend="rubric">ሩካን፡ እንድርያስ፡</hi><pb n="73ra"/> ወበርተሎሜዎስ፡ ሐዋርያቲሁ፡
+                          ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ እለ፡ ገብሩ፡ ውስተ፡ ሀገረ፡ በርቶስ፡ ወእምድኅረ፡ ግብአቶሙ፡
+                          እምሀገረ፡ ሳዎ፡ ዘይብልዋ፡ አልዋኅ፡ በሰላመ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ለዓለም፡ አሜን፨ ወኮነ፡
+                          እምድኅረ፡ ተንሥኦቱ፡ ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ እማእከለ፡ እሙታን፡ ወንጉሥ፡
+                          ውእቱ ፡ ላዕለ፡ ሰማይ፡ ወምድር፡ ወአስተርአዮ፡ ለበርተሎሜዎስ፡ በውስተ፡ ሀገረ፡
+                          አሕዛብ፡ ውስተ፡ በሓውርተ፡ አቅጥራን፡ ዘይእቲ፡ ሀገረ፡ አዝርዮስ፨
+                        </incipit>
+                        </msItem>
+
```
Comment (eu-genia) [inline ES/ESmy001.xml:313]: wrong nesting, delete the second closing /msItem tag

File: ES/ESmy001.xml [hunk 2]
```diff
@@ -30,324 +30,712 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                      እግዚእነ፡ አሜን፨ ወኮነ፡ ሶበ፡ ወፅአ፡ ቶማስ፡ ሐያርያ፡ ኀበ፡ ሀገረ፡ ህንዳኬ፡ ወሰበከ፡ ውስቴቶሙ፡ በአሚን፡
+                      ወዜነዎሙ፡ ሕገ፡ ወንጌል፡ ቅዱስ፨
+                    </incipit>
+                    <explicit>
+                    </explicit>
+                    <note>Ending <foreign xml:lang="gez">ወናሁ፡ ፈቀደ፡ ከመ፡ ሰማዕክሙ፡ እምኔየ፡ ተዓቀቡ፡ ወኢታርእዩ፡ ፍኖቶ፡ ለሰይጣን፡ ውስተ፡ አልባቢክሙ፡ ከመ፡
+                    ኢይቅረብ፡ ኀቤክሙ፡ ወኩኑ፡ ንቁሃነ፡ ትጽንሑ፡ ዳግመ፡ ምጽአቶ፡ ለእግዚእነ፡ ከመ፡ ይትመጦክሙ፡ ውስተ፡ መንግሥቱ፡
+                    ዘበሰማ<supplied reason="lost" resp="DN">ያት᎓</supplied></foreign>.</note>
+                 </msItem>
+
```
Comment (eu-genia) [inline ES/ESmy001.xml:582]: add a second closing /msItem tag here

File: ES/ESmy001.xml [hunk 3]
```diff
@@ -30,324 +30,712 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                              <item xml:id="q11" n="11">
                                  <dim unit="leaf">4</dim>
-                                 <locus from="102r" to="105v"/> XI፲፩(4/fols. 102r-105v) </item>
-                              <item xml:id="q12">
+                                 <num value="11">፲፩</num>
+                                 <locus from="102r" to="105v"/>
+                               </item>
+
+                              <item xml:id="q12" n="12">
                                  <dim unit="leaf">5</dim>
```
Comment (eu-genia) [inline ES/ESmy001.xml:674]: for quires with uneven number of leaves (12, 13, 14) you should specify which leaf is single

File: ES/ESmy001.xml [hunk 4]
```diff
@@ -363,129 +751,143 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                            <note/>
                            <ab type="ruling"/>
```
Comment (eu-genia) [inline ES/ESmy001.xml:752]: delete empty elements

File: ES/ESmy001.xml [hunk 5]
```diff
@@ -363,129 +751,143 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                           <ab type="pricking"> Primary pricks are partly visible (s. <locus target="#43r">43r</locus>).</ab>
-                           <ab type="pricking"> Ruling pricks are visible.</ab>
-                           <ab type="ruling"> The upper line is written below the ruling.</ab>
-                           <ab type="ruling"> The bottom line is written above the ruling.</ab>
+                           <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C.</ab>
+                           <ab type="pricking">Primary pricks are partly visible (see <locus target="#43r">43r</locus>).</ab>
+                           <ab type="pricking">Ruling pricks are visible.</ab>
+                           <ab type="ruling">The upper line is written below the ruling.</ab>
+                           <ab type="ruling">The bottom line is written above the ruling.</ab>
                         </layout>
```
Comment (eu-genia) [inline ES/ESmy001.xml:759]: delete `</layout><layout>` (lines 759-760), the layout description continues

File: ES/ESmy001.xml [hunk 6]
```diff
@@ -363,129 +751,143 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
                         <layout>
                            <ab type="punctuation" subtype="Executed">not quite regularly;</ab>
                            <ab type="punctuation" subtype="Usage">not always logical.</ab>
                            <ab type="punctuation">
```
Comment (eu-genia) [inline ES/ESmy001.xml:763]: ` <ab type="punctuation" subtype="Dividers">`

File: ES/ESmy001.xml [hunk 7]
```diff
@@ -363,129 +751,143 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
                               <list>
-                                 <item xml:id="div1"> Dividers:</item>
-                                 <item xml:id="div2"> I. Colon. </item>
-                                 <item xml:id="div3"> II. Four dot asterisk (e.g. fols.: 18va,
+                                 <item xml:id="div1">Dividers:</item>
```
Comment (eu-genia) [inline ES/ESmy001.xml:765]: remove item line 765 (`<item xml:id="div1">Dividers:</item>`) - the subtype dividers should be added to ab above

File: ES/ESmy004.xml [hunk 8]
```diff
@@ -30,360 +30,665 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                        ተርጐሞ፡ ለዘሰምዐ፡ ቃላቲሁ፡ በአሚን፡ ወበርትዕ፡ ሃይማኖት፡ ሕቡረ፡ [n.l.]ሐሮሙ፡ እግዚአብሔር፡ በመንግሥተ፡
-                        ሰማያት፡ ለዓለም፡ አሜን፨<foreign xml:lang="en">
-                           <locus target="#182va">182va</locus>, l.30 (Dionysius on the tribes of
-                           the Apostles): … And thanks-giving [be] to the Holy Spirit Who gave them
-                           knowledge and understanding so that [n.l.] in all [n.l.] the world
-                           concerning the Holy Trinity, one [God], to Whom [be] glory and honour and
+                           people. Forever and ever. Amen.
+                        </foreign>
+                            </incipit>
+
```
Comment (eu-genia) [inline ES/ESmy004.xml:66]: nesting problem, closing /msItem tag is missing here

Comment (eu-genia) [reply ES/ESmy004.xml:66]: (and there is then one too many at the end)

File: ES/ESmy004.xml [hunk 9]
```diff
@@ -30,360 +30,665 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                               </item>
+                              <item xml:id="q16" n="16">
                                  <dim unit="leaf">10</dim>
                                  <locus target="#10"/>
-                                 <locus from="113r" to="122v"/> XVI(10/fols. 113r-122v) </item>
-                              <item xml:id="q17">
+                                 <locus from="113r" to="122v"/>
+                               </item>
+                              <item xml:id="q17" n="17">
                                  <dim unit="leaf">7</dim>
```
Comment (eu-genia) [inline ES/ESmy004.xml:632]: specify which one is single, or it won't work

File: ES/ESmy004.xml [hunk 10]
```diff
@@ -396,117 +701,154 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                           <ab type="ruling">The upper written line is placed above, but on some folia below the ruling
+                             (see, e.g., <locus target="#62r #68v #69r #88v #89r #97v #98r"/>).</ab>
+                             <ab type="ruling">The bottom line is written above the ruling.</ab>
+                           <ab type="ruling">On some folia the scribe did not use one or two ruled lines at the bottom.</ab>
+
                         </layout>
                         <layout>
                            <ab type="punctuation" subtype="Executed">not quite regularly;</ab>
                            <ab type="punctuation" subtype="Usage">not always logical.</ab>
                            <ab type="punctuation">
```
Comment (eu-genia) [inline ES/ESmy004.xml:720]: subtype="Dividers"

File: ES/ESmy004.xml [hunk 11]
```diff
@@ -396,117 +701,154 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                                 <item xml:id="div3"> II. Colon with dashes above and below (esp.
-                                       <locus target="#79ra">79ra</locus>, l.17);</item>
-                                 <item xml:id="div4"> red and black dot vertical colon with dashes
-                                    above and below (esp. fols.: 27rb, l.9; 30va, l.2; 124rb, l.13;
-                                    137rb, l.6);</item>
-                                 <item xml:id="div5"> four dot asterisk (e.g. fols.: 14ra, ls. 9,
-                                    12; 21vb, l.1; 27rb, ls. 1, 4, 6; 36ra, l.3); </item>
-                                 <item xml:id="div6"> nine dot asterisk (used throughout the text). </item>
-                                 <item xml:id="div7"> Sentence dividers are accompanied by the
+                                 <item xml:id="div1">Dividers:</item>
```
Comment (eu-genia) [inline ES/ESmy004.xml:722]: remove the div1 line

File: ES/ESmy004.xml [hunk 12]
```diff
@@ -30,360 +30,665 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                           the Gadla ḥawāryāt of<ref type="mss" corresp="ESmy004">MY-004</ref>would
-                           belong to the IV recension’s group (cp. BauGaH, 83-84). Text I-3, the
-                           "Martyrdom of St Peter", does not contain the "Discourse of Peter on the
-                           Cross" (= BConAp I, 45-49; BConAp II, 30-34). </p>
-                     </note>
-                  </msItem>
-               </msContents>
+                   </msItem>
+                     </msItem>
+                    </msItem>
```
Comment (eu-genia) [inline ES/ESmy004.xml:544]: remove

File: ES/ESmy001.xml [hunk 13]
```diff
@@ -363,129 +751,143 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                                 <item xml:id="div10"> Crux ansata with coronis, marking the
-                                    beginning of each new text (used thoughout the Ms.).</item>
+                                    l.1; 126rb, l.8; 150rb, l.25).</item>
+                                 <item xml:id="div10">Crux ansata with coronis, marking the
+                                    beginning of each new text (used thoughout).</item>
                               </list>
                            </ab>
-                           <ab type="CruxAnsata"> </ab>
-                           <ab type="coronis"> </ab>
+                           <ab type="CruxAnsata"/>
```
Comment (eu-genia) [inline ES/ESmy001.xml:790]: are the elements present? if yes maybe better add locus?

**Review by eu-genia — CHANGES_REQUESTED** [review]
I am not sure as to what you are asking concerning the misplaced leaves - we usually regroup them in the description as you had done it for Ethio-SPaRe (see also recently https://github.com/BetaMasaheft/Documentation/issues/1770). Or of course you can add a specification of the relation in the description. 
After correcting and merging all changes - my advise is to delete your local copy of MSS folder and reclone it, to stop merging the old history every time.

File: ES/ESmy001.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy001.xml:313]: ok

File: ES/ESmy001.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy001.xml:582]: done

File: ES/ESmy001.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy001.xml:674]: ok

File: ES/ESmy001.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy001.xml:759]: ok

File: ES/ESmy001.xml (hunk 7 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy001.xml:765]: like this?

Comment (DenisNosnitsin1970) [reply ES/ESmy001.xml:765]: The punctuation signs will not be in the catalogue.

File: ES/ESmy001.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy001.xml:674]: Here, I comment out the collation; it is very hypothetical and should better not be presented in the catalogue.

File: ES/ESmy001.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy001.xml:763]: done?

File: ES/ESmy004.xml (hunk 9 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy004.xml:632]: ok

File: ES/ESmy004.xml (hunk 8 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy004.xml:66]: ok

File: ES/ESmy004.xml (hunk 12 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy004.xml:544]: ok

File: ES/ESmy001.xml (hunk 13 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy001.xml:790]: done

File: ES/ESmy004.xml (hunk 11 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy004.xml:722]: ok

File: ES/ESmy001.xml (hunk 7 above)
Comment (eu-genia) [reply ES/ESmy001.xml:765]: no, it should be 
<ab type="punctuation" subtype="Dividers">
                              <list>
                                 
                                 <item xml:id="div1"> I. Colon</item>
<item xml:id="div2"> II. Four dot asterisk ...........................</item>
</ab>

Comment (DenisNosnitsin1970) [reply ES/ESmy001.xml:765]: Hope now correct.

**Review by eu-genia — APPROVED** [review]

**Comment by eu-genia** [comment]
my004 not yet pushed?

File: ES/ESmy004.xml [hunk 14]
```diff
@@ -631,6 +631,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                               <item xml:id="q17" n="17">
                                  <dim unit="leaf">7</dim>
                                  <locus from="123r" to="129v"/>
+                                 s.l.: 7, no stub
```
Comment (eu-genia) [inline ES/ESmy004.xml:634]: if you do not specify which one is sl then the app cannot work for the diagrams and formulas

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ES/ESmy004.xml (hunk 14 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy004.xml:634]: But it is specified, s.l.: 7, no stub. No?

Comment (eu-genia) [reply ES/ESmy004.xml:634]: sorry yes you are right it is the `<note>` that. at least in the preview, causes problems - it works if you move the note above the locus line

**Review by eu-genia — APPROVED** [review]

**Pull request merged**